tests/integers.rs

```diff
@@ -65,14 +87,28 @@ fn add_windows_around_points(TestCase { min, max }: TestCase, points: Points) ->
 }
 
 /// Test the variable size integer encoding
+/// Additionally, it outputs the byte array to integer value mapping like: https://github.com/janestreet/bin_prot/blob/5915cde59105f398b53f682c5f4dad29e272f696/test/integers_repr_tests_64bit.ml
 #[test]
 fn test_roundtrip_integers() {
     let int_test = TestCase {
         min: INT_MIN,
         max: INT_MAX,
     };
 
+    let mut f = std::fs::OpenOptions::new()
+        .truncate(true)
+        .create(true)
+        .write(true)
+        .open("tests/integers_repr.bin")
+        .unwrap();
+    f.write(
+        "// WARNING: never accept the corrected output for this file, it must never change!\n\n\n"
+            .as_bytes(),
+    )
+    .unwrap();
```

Comment:
does this create a new file if it doesn't exist, or open it if it does exist?

Reply:
It's suppose to create the new file every time the test is run